doc/src/properties/residue.toml

```diff
@@ -25,6 +25,8 @@ PDB = """The **chainname** defines the name assigned to a protein chain by
 biologists. It is is a single character used to group residues which are linked
 together in a crystallographic file where it may not be unique."""
 
+PSF = """ TODO """
```

Comment:
@ShoubhikRaj could you help me here? The chain name is set by the molfile plugin to be the first letter of the segment name, does it make sense to expose it to the users?

Reply:
@Luthaf I don't do much work with biomolecules either. I have never needed to use the chain name. I think if molfiles reads it, there is no reason not to expose it to the users. If there are no biomolecules, then I don't know how much chainname is necessary. For example, the segName for water is TIP4P for all TIP4P type water systems, and getting chainname T is not particularly informative.

Reply:
>  Can it be different for other formats that chemfiles accesses via molfiles?

Not now, but maybe in the future if we add other molfiles format. This "the first letter of segname is chainname" behavior is specific to the PSF reader.

I've edited the documentation to mention that it is always the first letter, this way the format works in the same way as most biomolecules formats.